bundles/org.openhab.binding.energidataservice/README.md

```diff
@@ -605,6 +668,16 @@ console.log("Spot price two hours from now: " + price);
 
 :::
 
+::: tab JRuby
+
+```ruby
+two_hours_from_now = 2.hours.from_now.truncated_to(ChronoUnit::HOURS)
+price = SpotPrice.persisted_state(two_hours_from_now)
```

Comment:
where does `SpotPrice` come from?

Reply:
It's an item, as per the examples from the other languages

Reply:
🤦  I was thinking it was an Action, and was wonder how the constant got imported into Ruby, since we don't put them in the top level namespace.